task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -0,0 +1,237 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: build-vm-image
+spec:
+  params:
+    - description: The platform to build on
```

Comment:
An example would be helpful

Reply:
Ah, I agree - but this is just as it is for `task/buildah-remote/`. The description string is the same.